src/operator/batch_norm_v1.cc

```diff
@@ -89,6 +89,9 @@ the output. It is often used during inference.
 Both ``gamma`` and ``beta`` are learnable parameters. But if ``fix_gamma`` is true,
 then set ``gamma`` to 1 and its gradient to 0.
 
+There's no sparse support for this operator, and will exhibit problematic behavior if used with
```

Comment:
Nit: "There's no sparse support for this operator, and will" -> "There's no sparse support for this operator. It will"